ptp/writer.go

```diff
@@ -2,13 +2,55 @@ package ptp
 
 import (
 	"bufio"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"math"
 	"os"
 )
 
+type Point struct {
```

Comment:
Can we move `Point`, `BoundingBox`, and `Summary` to a new file called `summary.go`? This file's already got a lot going on in it

Reply:
Also, it looks like the `json` tags aren't needed/used since we have internal structs in the `Save` method